codeland.html

```diff
@@ -1,12 +1,101 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>CodeLand: Distributed</title>
-</head>
-<body>
-    <div>CodeLand: Distributed.</div>
-    <div>Coming soon.</div>
-</body>
-</html>
+<style>
+  /* Mobile first */
+  .codeland-content {
+    background-color: #E5E5E5;
+    height: 100%;
+    padding-top: 30px;
+  }
+
+  .codeland-row {
+    width: 100%;
+    padding: 0;
+    margin-top: 25px;
+    flex-wrap: wrap;
+  }
+
+  .h-flex {
+    display: flex;
+  }
+
+  .v-flex {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
We can eventually throw all of this away... This is just a poor attempt at having some structure for now